ReportPortal.Client/Requests/FinishTestItemRequest.cs

```diff
@@ -36,14 +37,17 @@ public DateTime EndTime
         public string StatusString { get { return EnumConverter.ConvertFrom(Status); } set { Status = EnumConverter.ConvertTo<Status>(value); } }
 
         public Status Status = Models.Status.Passed;
-
-        [DataMember(Name = "retry", EmitDefaultValue = false)]
```

Comment:
Regarding api docs FinishTestItemRequest has Retry property too. Could you please return it back?

Reply:
done